content/en/account_management/rbac/data_access.md

```diff
@@ -63,6 +65,33 @@ The Data Access Control API is under development and should be considered unstab
 
 Terraform support will be announced after Data Access Control is generally available.
 
+### Supported telemetry types {#supported-telemetry}
+
+- APM traces
+- CI Visibility pipelines
+- Cloud costs
+- Custom metrics
+    - **Note:** Standard metrics are not supported
+- Error Tracking issues
+- Logs
+- RUM sessions
+
+## Usage constraints
+
+After you turn on Data Access Control, Datadog disables or limits other features to control access to sensitive data. See the list of affected features below to see how they are restricted.
+
+### Real User Monitoring (RUM)
+
+#### Session Replay: Extended Retention
+By default, Session Replay data is retained for 30 days. To extend retention to 15 months, you can enable Extended Retention on individual session replays. When you enroll in the preview for Data Access Control, Datadog disables the option for Extended Retention. Extended Retention does not work with Data Access Control because the data store that holds the extended data does not support access restrictions.
```

Comment:
```suggestion
By default, Session Replay data is retained for 30 days. To extend retention to 15 months, you can enable Extended Retention on individual session replays. When you enroll in the Preview for Data Access Control, Datadog disables the option for Extended Retention. Extended Retention does not work with Data Access Control because the data store that holds the extended data does not support access restrictions.
```

Reply:
It looks like we mostly capitalize this when referring to product previews